src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const ONE_DAY_PRICE = 40;
+  const BASE_PRICE = days * ONE_DAY_PRICE;
```

Comment:
It would be better to keep simple constants all together, and this one with calculation somewhere underneath